x-pack/filebeat/module/zeek/README.md

```diff
@@ -0,0 +1,73 @@
+# Zeek (Bro) module
+
+## Caveats
+
+* Module is to be considered _alpha_.
+* Field names will be changing for 7.0 to comply with Elastic Common Schema (ECS).
+
+## How to try the module from source
```

Comment:
Great tutorial, thanks for writing it! I think that after we have the module in a released version, we might want to switch to use the downloaded binaries, which should make it a bit easier.

Reply:
Maybe I can rename this to README-developer.md, and introduce a README.md for normal users that use binary distributions.